src/sentry/demo/demo_start.py

```diff
@@ -250,3 +263,16 @@ def _get_one_transaction_name(project: Project):
         referrer="sandbox.demo_start._get_one_transaction_name",
     )
     return result["data"][0]["transaction"]
+
+
+def check_strings_similar(error_type, group):
+    type = group.data["metadata"].get("type")
+    title = group.data["metadata"].get("title")
+    checks = []
+    if type:
+        type = type.lower()
+        checks.append(error_type in type or error_type == type)
```

Comment:
@roggenkemper You don't need the extra `or error_type == type` since `error_type in type` will be true

Reply:
Also, I don't think you even need `checks` as a variable. Can't you just do something like;
```
if error_type in type.lower():
   return True
```
then at the bottom just `return False` if it matches neither

Reply:
removed!